bin/linting.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
I am not familiar at all with node executable scripts; do we need a Linux variant of the script as well or how do things work?

Reply:
This is a kind of shim that I've seen created by JS tools - I believe it's only needed to be able to use them *globally* (i.e. outside of invocation via the package manager). After some experimenting it seems like package managers already create these shims themselves so doing it by hand is actually unnecessary.

So to sum up, only a node-executable script needs to be defined in `package.json -> bin` for it to be runnable.